README.md

```diff
@@ -206,6 +206,29 @@ var publicKey = fs.readFileSync('/path/to/public.pub');
 app.use(jwt({ secret: publicKey }));
 ```
 
+If the secret option is a function, this function is called for each JWT received in
```

Comment:
Sorry to be pedantic :-) This should be \`secret\` rather than secret. And `node-jwks-rsa` as a link, ie \[node-jwks-rsa\]\(https://github.com/auth0/node-jwks-rsa) so it looks like [node-jwks-rsa](https://github.com/auth0/node-jwks-rsa).

Once that is done, I think it is ready - thanks for your quick response this morning!